bin/cml/runner.js

```diff
@@ -333,23 +323,26 @@ const runLocal = async (opts) => {
 };
 
 const run = async (opts) => {
+  process.on('unhandledRejection', (reason) =>
+    shutdown({ ...opts, error: new Error(reason) })
```

Comment:
```suggestion
    shutdown({ ...opts, error: reason instanceof Error ? reason : new Error(reason) })
```

Reply:
Its not going to be an Error unless an Error is what is rejected (uncommon) 
https://developer.mozilla.org/en-US/docs/Web/API/Window/unhandledrejection_event

In any case for your curiosity 😉 
```js
const err = new Error('hi')
const err2 = new Error(err)
console.log(err2.message)
```

Reply:
https://nodejs.org/dist/latest-v16.x/docs/api/process.html#event-unhandledrejection 
- `reason`  [Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error)  |  [any](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Data_types) The object with which the promise was rejected (typically an [Error](https://nodejs.org/dist/latest-v16.x/docs/api/errors.html#class-error) object).